lib/src/message.dart

```diff
@@ -0,0 +1,199 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:collection/collection.dart';
+import 'package:http_parser/http_parser.dart';
+
+import 'body.dart';
+import 'headers.dart';
+import 'http_unmodifiable_map.dart';
+
+Body getBody(Message message) => message._body;
```

Comment:
Can do. It wasn't documented in `shelf` so I had assumed it was more of a utility function.